modules/apps/frontend-js/frontend-js-dropdown-support-web/src/main/resources/META-INF/resources/DropdownProvider.js

```diff
@@ -97,6 +106,13 @@ class DropdownProvider {
 		trigger.parentElement.classList.add(CssClass.SHOW);
 		trigger.setAttribute('aria-expanded', true);
 
+		document.addEventListener('mousedown', (event) =>
```

Comment:
We should have the same events for this and for the trigger, so if there are the right ones let's change the others ;)